packages/tezos-ts-utils/src/tezos-ts-utils.ts

```diff
@@ -0,0 +1,258 @@
+import { Buffer } from 'buffer';
+const bs58check = require('bs58check');
+
+export { prefix } from './contants';
+
+export function b58cencode(payload: string | Uint8Array, prefix: Uint8Array) {
+  const payloadAr =
+    typeof payload === 'string' ? Uint8Array.from(Buffer.from(payload, 'hex')) : payload;
+
+  const n = new Uint8Array(prefix.length + payloadAr.length);
+  n.set(prefix);
+  n.set(payloadAr, prefix.length);
+
+  return bs58check.encode(Buffer.from(n.buffer));
+}
+
+export const b58cdecode = (enc: string, prefixArg: Uint8Array): Uint8Array =>
+  bs58check.decode(enc).slice(prefixArg.length);
+
+export function b58decode(payload: string) {
+  const buf: Buffer = bs58check.decode(payload);
+  const buf2hex = function(buffer: Buffer) {
+    const byteArray = new Uint8Array(buffer);
+    const hexParts = [] as any[];
+    for (let i = 0; i < byteArray.length; i++) {
+      let hex = byteArray[i].toString(16);
+      let paddedHex = ('00' + hex).slice(-2);
+      hexParts.push(paddedHex);
+    }
+    return hexParts.join('');
+  };
+
+  const prefix = {
+    [new Uint8Array([6, 161, 159]).toString()]: '0000',
```

Comment:
Instead of using literals here, using constants with a descriptive name will make the code a lot more readable and easier to grok.